src/template.css

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Would avoid adding licensed code here.

Reply:
What do you suggest? Redesign the template?
It's the few selectors from bootstrap which we actually use in the template (before I included the full bootstrap CSS, but then the empty template was 238 KB)

Reply:
Assuming you will update the PR with a template similar to the one I've sent you.

Reply:
done